app/helpers/quadicon_helper.rb

```diff
@@ -641,6 +699,15 @@ def render_single_quad_quadicon(item, options)
     output.collect(&:html_safe).join('').html_safe
   end
 
+  def img_for_health_state(item)
```

Comment:
Agreed, better have it together with the 4 other methods, than in an unexpected place by itself.